components/section/data-section-loader.tsx

```diff
@@ -28,6 +33,14 @@ export default function DataSectionLoader({
           </ul>
         </>
       )}
-    </div>
+      {after5s && (
+        <FadeIn>
+          <Warning>
+            Le service semble particulièrement occupé en ce moment, cela peut
+            prendre encore un peu de temps (10s à 20s).
+          </Warning>
+        </FadeIn>
+      )}
+    </>
```

Comment:
For UX coherence I would add it before as it is the way it is displayed accross the app